packages/react/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -7028,6 +7028,9 @@ Map {
       "className": Object {
         "type": "string",
       },
+      "isRail": Object {
+        "type": "bool",
+      },
```

Comment:
With three more components needing `isRail`, is it worth considering using context to store the value of `isRail` and using that context to get the value within the components that need it? Otherwise we'll be relying on consumers to pass `isRail` to these components, right? I think it's unlikely that we'll see folks properly configuring this if that is the case.

Reply:
Good call! I've just moved the isRail prop from being spread through the children elements and added a Context to the `SideNav` @tay1orjones